neo/Prompt/Commands/Wallet.py

```diff
@@ -305,6 +306,66 @@ def command_desc(self):
         return CommandDesc('unspent', 'show unspent assets', params=[p1, p2, p3, p4])
 
 
+class CommandWalletSplit(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) < 4:
+            print("Please specify the required parameters")
+            return None
+
+        if len(arguments) > 5:
+            # the 5th argument is the optional attributes,
+            print("Too many parameters supplied. Please check your command")
+            return None
+
+        try:
+            from_addr = wallet.ToScriptHash(arguments[0])
+        except ValueError as e:
+            print(str(e))
+            return None
+
+        asset_id = PromptUtils.get_asset_id(wallet, arguments[1])
+
+        try:
+            index = int(arguments[2])
+        except ValueError:
+            print(f"Invalid unspent index value: {arguments[2]}")
+            return None
+
+        try:
+            divisions = int(arguments[3])
+        except ValueError:
+            print(f"Invalid divisions value: {arguments[3]}")
+            return None
+
+        if divisions < 1:
+            print("Divisions cannot be lower than 1")
+            return None
```

Comment:
I think the divisions should not be lower than 2. When I specified `1` it would not split anything. Specifying `2` did work.